src/components/TaskHeader/index.tsx

```diff
@@ -0,0 +1,39 @@
+import type { VFC } from "react";
+
+const ScheduleType = {
+  TODAY: "today",
+  TOMORROW: "tomorrow",
+  NEXT: "next",
+} as const;
+type ScheduleType = typeof ScheduleType[keyof typeof ScheduleType];
+
+type Props = {
+  schedule: ScheduleType;
+};
+
+type ThemeType = {
+  title: string;
+  color: string;
+};
+
+const TaskHeader: VFC<Props> = ({ schedule }) => {
+  const theme: Readonly<Record<ScheduleType, ThemeType>> = {
```

Comment:
```
Readonly<Record<ScheduleType, ThemeType>>
```
このセットそれ自体が ThemType なのではないかな？と思いました 🤔 
ので、内側の ThemeType は違う名目のほうがわかりやすいかと思いました。
（ミスリーディングしてしまいそう）

---

また、 ThemeType を外で型宣言して、
```suggestion
  const theme: ThemeType = {
```
としてあるとなおよさそうかなと思いました。